src/wallet/rpcwallet.cpp

```diff
@@ -430,6 +439,8 @@ static UniValue sendtoaddress(const JSONRPCRequest& request)
             "       \"" + FeeModes("\"\n\"") + "\""},
                     {"avoid_reuse", RPCArg::Type::BOOL, /* default */ "true", "(only available if avoid_reuse wallet flag is set) Avoid spending from dirty addresses; addresses are considered\n"
             "                             dirty if they have previously been used in a transaction."},
+                    {"verbose", RPCArg::Type::BOOL, /* default */ "false",
+                            "Optional, If true, return extra information about the transaction."},
                 },
                 RPCResult{
```

Comment:
The `RPCResult` will need to be updated to include the `verbose` result too.

Reply:
This isn't quite what I was looking for.

There are 2 possible results, so we need to have different RPC results for them. `getrawtransaction` has a similar `verbose` result. I would suggest looking at that to see how this should be done.